src/main/kotlin/tanvd/grazi/ide/ui/GraziSettingsPanel.kt

```diff
@@ -0,0 +1,98 @@
+package tanvd.grazi.ide.ui
+
+import com.intellij.ide.plugins.newui.VerticalLayout
+import com.intellij.openapi.options.ConfigurableUi
+import com.intellij.openapi.ui.ComboBox
+import com.intellij.ui.CheckBoxList
+import com.intellij.ui.components.JBCheckBox
+import com.intellij.ui.components.JBScrollPane
+import com.intellij.ui.components.JBTabbedPane
+import tanvd.grazi.GraziBundle
+import tanvd.grazi.GraziConfig
+import tanvd.grazi.GraziPlugin
+import tanvd.grazi.language.Lang
+import java.awt.*
+import javax.swing.*
+
+
+class GraziSettingsPanel : ConfigurableUi<GraziConfig> {
+
+    private val panelEmptyBorder = BorderFactory.createEmptyBorder(3, 5, 5, 5)
+
+    private val cbEnableGraziSpellcheck = JBCheckBox(GraziBundle.message("grazi.ui.settings.enable.text"))
```

Comment:
Hungarian notation, huh?)

Reply:
Old habit, sometimes you need to know what kind of element it is...